modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/apps/ListApps.es.js

```diff
@@ -90,19 +90,22 @@ export default ({
 	const newAppLink = compile(editPath[0])({dataDefinitionId, objectType});
 
 	const ADD_BUTTON = () => (
-		<Button
-			className="nav-btn nav-btn-monospaced"
-			href={newAppLink}
-			symbol="plus"
-			tooltip={Liferay.Language.get('new-app')}
-		/>
+		<Link to={newAppLink}>
```

Comment:
Hello, @kevenleone!

You are correct about history.push, I will adjust this. Tooltip Provider has been removed because the component uses the Liferay Portal provider.